templates/jaasai/community.html

```diff
@@ -141,4 +151,5 @@ <h5>Social</h5>
   </div>
 </div>
 
+<script src="{{ url_for('static', filename='js/dist/blog-feed.min.js') }}"></script>
```

Comment:
Does this have cache busting?

Reply:
It does not. I've added a task (https://github.com/canonical-websites/jaas.ai/issues/164) to add this across the app.